gatorgrader.py

```diff
@@ -6,6 +6,8 @@
 
 
 if __name__ == "__main__":
+    # assume that the checks pass and prove otherwise
+    exit_code = 0
     # orchestrate check(s) of the specified deliverable(s)
     exit_code = orchestrate.check(sys.argv[1:])
```

Comment:
I agree with this point and I have fixed the code. The reason a separate assignment was in the code was because I used it previously during debugging when there was code _between_ the assignment of the `exit_code` and the call to the `orchestrate` call. I think that we can safely remove the assignment, which I have tested locally and confirmed that it worked.

Reply:
I am going to mark this issue as resolved, let me know if there are points about it that I should handle, okay?